e3db/client.py

```diff
@@ -1,12 +1,23 @@
 from auth import E3DBAuth
-from crypto import Crypto
+import os
+if 'CRYPTO_SUITE' in os.environ and os.environ['CRYPTO_SUITE'] == 'NIST':
+    from nist_crypto import NistCrypto as Crypto
+else:
+    from sodium_crypto import SodiumCrypto as Crypto
 from config import Config
 from types import ClientDetails, ClientInfo, IncomingSharingPolicy, OutgoingSharingPolicy, Meta, QueryResult, Query, Record
 from exceptions import APIError, LookupError, CryptoError, QueryError, ConflictError
 import requests
 import uuid
 
 
+def crypto_mode():
+    if 'CRYPTO_SUITE' in os.environ and os.environ['CRYPTO_SUITE'] == 'NIST':
```

Comment:
Should just export this value from `Crypto`.

Reply:
Good call, and will be an easy change.